examples/incubating.md

```diff
@@ -1292,7 +1292,7 @@ displayed on the light grey of tables.*
     </tr>
 </table>
 
-#### Notary Logos
+#### NotaryProject Logos
```

Comment:
```suggestion
#### Notary Project Logos
```

Reply:
Please also help update the "Notary" to "Notary Project" in README file https://github.com/cncf/artwork?tab=readme-ov-file#cncf-logos. Thanks